modules/solid_properties/test/tests/functional/test.i

```diff
@@ -0,0 +1,115 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+[]
+
+[Variables]
```

Comment:
Mind doing a global replace of

`[../]` -> `[]` and `[./` -> `[`?

Reply:
Yep. These files were from back in ~2018 so there's some old stuff hanging around